test/PlaygroundTransform/placeholder.swift

```diff
@@ -3,7 +3,14 @@
 // RUN: %target-build-swift -Xfrontend -playground -Xfrontend -disable-playground-transform -o %t/main %t/main.swift
 // RUN: %target-codesign %t/main
 // RUN: %target-run %t/main | %FileCheck %s
-// RUN: ! %target-run %t/main --crash 2>&1 | %FileCheck -check-prefix=CRASH-CHECK %s
+// RUN: %{python} -c "import sys; import subprocess; exec(\"\"\"\n\
+// RUN:try:\n\
+// RUN:    print(subprocess.check_output('%target-run %t/main --crash', stderr=subprocess.STDOUT, shell=True))\n\
```

Comment:
You should be able to use `check_call` here to avoid having to buffer output.

Reply:
@jrose-apple you mean check_call instead of check_output ?

Moved the py code into a separate file. Any other suggestions?